app/examples/native-module/README.md

```diff
@@ -0,0 +1,16 @@
+## Native modules example
```

Comment:
Please add a brief summary of what is this example all about and how is it useful.